jcl/jpp_configuration.xml

```diff
@@ -470,4 +470,18 @@
 		<parameter name="jxerules:outputdir" value="java/lang"/>
 	</configuration>
 
+	<configuration
+		  label="SIDECAR18-TOOLS-OPENJ9"
+		  outputpath="SIDECAR18-TOOLS/src"
+		  dependencies="SIDECAR18-SE-OPENJ9"
```

Comment:
This dependency makes `SIDECAR18-TOOLS-OPENJ9` includes flag `CUDA4J,DAA` which is not necessary for this tool pconfig.

Reply:
Agreed, but it didn't seem harmful. Do you prefer to use SIDECAR18-SE as the dependency and then declare the flag Sidecar18-SE-OpenJ9? Using SIDECAR18-SE-OPENJ9 as the dependency means we automatically pick up any future changes to SIDECAR18-SE-OPENJ9.

Reply:
The flag `Sidecar18-SE-OpenJ9` supposedly picks up any future changes to `OpenJ9 Java 8 `. Using SIDECAR18-SE-OPENJ9 as the dependency can get any flag changes in that configration.
Agreed that it won't hurt to include flag `CUDA4J DAA`.